src/component/SearchBar.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import {Button} from 'semantic-ui-react';
+
+class SearchBar extends React.Component {
+
+   state = {
+      term: ''
+   };
+
+   onLogOutClick = event =>{
+
+      localStorage.removeItem('user');
+      this.props.history.push('/');
+   }
+
+   onFormSubmit = event => {
+      event.preventDefault();
+      this.props.onFormSubmit(this.state.term);
+   }
+
+   render() {
+      return (
+         <form onSubmit={this.onFormSubmit}>
+            <div className="ui menu">
+               <div className=" ui header item">YouTube Video Search</div>
```

Comment:
@RajanTank Extract all your lables in a separate file, @ShaileshPrajapati-BTC will show the approach.

Reason is - What if you want to change everything to french? OR you want to change some lables? It would be easier to modify them quickly.